terminal/display.go

```diff
@@ -1,9 +1,25 @@
 package terminal
 
+import (
+	"fmt"
+)
+
 type EraseLineMode int
+type EraseScreenMode int
 
 const (
 	ERASE_LINE_END EraseLineMode = iota
 	ERASE_LINE_START
 	ERASE_LINE_ALL
 )
+
+const (
+	ERASE_SCREEN_END EraseScreenMode = iota
+	ERASE_SCREEN_START
+	ERASE_SCREEN_ALL
+)
+
+func EraseScreen(out FileWriter, mode EraseScreenMode) error {
+	_, err := fmt.Fprintf(out, "\x1b[%dJ", mode)
```

Comment:
on Windows we can't just assume that printing `\x1b[%dJ` to the terminal will just work. I'm pretty sure that out-of-the-box, that won't work in both cmd.exe and PowerShell.exe terminal, and seeing how Survey right now works in both, we would have to come up with a solution to clear to the end of the terminal that would work in all terminals that Survey currently supports.

Reply:
Completely agree! I'd be interested in knowing if a modified version of [this example](https://learn.microsoft.com/en-us/windows/console/clearing-the-screen#example-3) could be used to clear only from the current cursor position to the end of the screen? I have hope that the `FillConsoleOutputCharacter` call is buffered and would fill the remaining screen with blanks in a single action, which should mean that option flickering is removed from select prompts on Windows, but I am unfamiliar with these calls and am open to any suggestions!